modelci/app/readme.md

```diff
@@ -3,7 +3,7 @@
 ## Start server
 
 ```shell script
-uvicorn main:app
+uvicorn main:app --port 8000
```

Comment:
Default is 8000?

Reply:
Yes, according to https://www.uvicorn.org/settings/#socket-binding, default port is 8000.